pkg/utils/filter_map.go

```
@@ -0,0 +1,34 @@
+/*
+Copyright © 2020 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package utils
+
+import (
+	"strings"
+)
+
+// FilterMap returns a new map[string]T containing origMap keys that match aginst the keysToSelect
```

nit: Could we include in this comment that if a key in `keysToSelect` does not exist in the `origMap`, it will be silently ignored?